frontend/ui/elements/platform_navigation.lua

```diff
@@ -0,0 +1,36 @@
+local Device = require("device")
+local Event = require("ui/event")
+local UIManager = require("ui/uimanager")
+local _ = require("gettext")
+
+-- This whole menu is hidden behind a hasKeys device cap.
+local PlatformNav = {
+    text = _("Page turn behavior"), -- Mainly so as to differentiate w/ "Page Turns" when in readermenu...
```

Comment:
The name of that file, the PlatformNav abbreviated object name :) and that menu item text feels a bit unrelated and odd to me.
Page turn buttons and key repeats feel quite far from "page turn behavior".
No real idea about a better alternative though.

Reply:
Yeah, same boat as the naming concerns above, although I did try a bit harder on that one ;p.